rollup.config.ts

```diff
@@ -0,0 +1,70 @@
+import * as fs from 'fs'
+import type { RollupOptions } from 'rollup'
+import jsonPlugin from '@rollup/plugin-json'
+import nodeResolvePlugin from '@rollup/plugin-node-resolve'
+import typescriptPlugin from '@rollup/plugin-typescript'
+import terserPlugin from '@rollup/plugin-terser'
+import dtsPlugin from 'rollup-plugin-dts'
+import licensePlugin from 'rollup-plugin-license'
+import terserConfig from './terser.config'
+
+const { dependencies } = JSON.parse(fs.readFileSync('package.json', 'utf8'))
+
+const outputDirectory = 'dist'
+
+export const commonInput = {
+  input: 'src/index.ts',
+  plugins: [nodeResolvePlugin(), jsonPlugin(), typescriptPlugin()],
+}
+
+export const commonOutput = {
+  name: 'BotD',
+  exports: 'named' as const,
+  plugins: [
+    licensePlugin({
+      banner: {
+        content: {
+          file: 'resources/license_banner.txt',
+        },
+      },
+    }),
+  ],
+}
+
+const commonTerser = terserPlugin(terserConfig)
```

Comment:
Terser is not used in this project. Please remove all mentions of Terser from the “green” code.